### ISSUE TITLE
Show only online or offline but no %

### ISSUE THREAD

**Issue by Skyfay** [louislam/uptime-kuma#1028]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

I haven't found a way to only display online or offline on the / status page.
I want to see the% numbers on the dashboard but not on the status page.
Like in this picture:
![image](https://user-images.githubusercontent.com/76796007/145384082-a4747dc1-8177-4c5a-9fb3-27f18fd7b328.png)
That would be very cool!

### ✔️ Solution

I want that because I just want to show the users who are watching whether online or offline and not the percentages.

**Comment by NixNotCastey** [comment]
It's probably should be an option like "Show uptime pills/percentage on status page - ON/OFF"

**Comment by Skyfay** [comment]
> It's probably should be an option like "Show uptime pills/percentage on status page - ON/OFF"

Yes.  that would be very cool if you could enable or disable that.

**Comment by Zellmer** [comment]
so should there be shown what the last status was?

**Comment by Skyfay** [comment]
> 

Yes

**Comment by Skyfay** [comment]
> so should there be shown what the last status was?

Yes

**Comment by Zellmer** [comment]
ok i'm new but i would like to take a closer look at it. At first glance, we would have to add a value to the monitor_group table in the database (maybe ShowPercent or something).

**Comment by Zellmer** [comment]
![image](https://user-images.githubusercontent.com/23472788/149787170-1b413096-c05f-4c93-9f0e-5ea83de57cef.png)

**Comment by Skyfay** [comment]
> Ok First Step Done

looks awesome!

**Comment by karelkryda** [comment]
Hi @Zellmer, are you still working on it? 🙂

**Comment by Zellmer** [comment]
Im stuck at the database

**Comment by karelkryda** [comment]
If you add a "showPercentage" column or something similar to each monitor in the database and add a checkbox to create/edit the monitor, it will allow you to simply display the information in the right way for each monitor on the status page.

**Comment by Zellmer** [comment]
Yes, I know I intend to do that too, but I don't know where to add the column and how to get the data

**Comment by Skyfay** [comment]
How does it look here, is there already new information? I would really appreciate this feature.
